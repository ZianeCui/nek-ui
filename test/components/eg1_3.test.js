import { shallow } from 'vue-test-utils'
import Test2 from '../../components/Example/eg1/test2.vue'

describe('使用shallow测试组件', () => {
    let cmp

    beforeEach(() => {
        cmp = shallow(Test2, { // Create a shallow instance of the component
            data: {
                messages: ['Cat']
            }
        })
    })

    it('messages的值等于["Cat"]', () => {
        // Within cmp.vm, we can access all Vue instance methods
        expect(cmp.vm.messages).toEqual(['Cat'])
    })

    it('eg1_test2组件快照测试', () => {
        expect(cmp.element).toMatchSnapshot()
    })
})