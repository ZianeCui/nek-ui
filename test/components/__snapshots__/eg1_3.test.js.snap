// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`使用shallow测试组件 eg1_test2组件快照测试 1`] = `
<div
  id="app"
>
  <!---->
</div>
`;
